services/service.ts

```diff
@@ -40,13 +40,15 @@ const homebrewBaseDir = ((): string | null => {
 })();
 
 // Maps internal setting field name with filesystem flag name.
-type FlagName = string;
-const availableFlags = {
+// TODO: Figure out how to avoid repeating these.
```

Comment:
Something like this seems to pass all the checks.
```typescript
const availableFlags = {
  telnetDisabled: 'webosbrew_telnet_disabled',
  failsafe: 'webosbrew_failsafe',
  sshdEnabled: 'webosbrew_sshd_enabled',
  blockUpdates: 'webosbrew_block_updates',
} as const;
type FlagName = keyof typeof availableFlags;
type FlagFileName = (typeof availableFlags)[FlagName];
```

Reply:
I could have sworn I tried that...